bats/sql-shell.expect

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/expect
+
+set timeout 1
+spawn dolt sql
+set id $spawn_id
+
+expect -i id "doltsql> "
+send -i id "CREATE TABLE test_expect (pk int primary key);\r"
+
+expect -i id "doltsql> "
+
+# spawn the second process
+spawn dolt sql
+set id2 $spawn_id
+
+# Todo: Should this be a dolt ls instead ?
+expect -i id2 "doltsql> "
+send -i id2 "show tables;\r"
+
```

Comment:
You're not actually examining the output of show tables here

Reply:
It should get examined in the test case

Reply:
You can't examine it in the test case, that's too late. The whole point is to make sure that the two SQL sessions can read each other's writes before they exit. In the bats script, both processes have exited by the time you do any assertions. It's too late.